src/main/java/com/elixr/poc/rest/controller/UserRetrievalController.java

```diff
@@ -45,4 +46,15 @@ public ResponseEntity<UserResponse> retrieveUser(@PathVariable("userId") @Valid
                 .firstName(user.getFirstName()).lastName(user.getLastName()).build();
         return new ResponseEntity<>(userResponse, HttpStatus.OK);
```

Comment:
You should handle not found cases too

Reply:
It is getting handled by GlobalExceptionHandler class